compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/functionexpr-conditional-access-2.tsx

```diff
@@ -0,0 +1,12 @@
+// @enableTreatFunctionDepsAsConditional
+import { Stringify } from "shared-runtime";
+
+function Component({ props }) {
```

Comment:
nit: it's confusing to destructure props and name the identifier `props`, pick a different name?